DigitalPassaport/DigitalPassaport/Epics/CreateAccount/ViewModel/CreateAccountViewModel.swift

```diff
@@ -18,7 +18,7 @@ class CreateAccountViewModel {
     }
     
     func createAccountButtonTapped(completion: @escaping (Bool) -> Void) {
-        repository.fetchData(parameters: nil)
+        repository.fetchData()
```

Comment:
I'm not sure what the point of this is. Why do you handle events and then do nothing here... Just need to sink

Reply:
Here is removed handleEvents